openassessment/fileupload/backends/filesystem.py

```diff
@@ -0,0 +1,108 @@
+
+from .base import BaseBackend
+from .. import exceptions
+
+from django.conf import settings
+import django.core.cache
+from django.core.urlresolvers import reverse
+
+
+class Backend(BaseBackend):
```

Comment:
Since you added `get_backend()` that code will call to instantiate the correct backend, perhaps you should name this `LocalFileBackend` and the the S3 `Backend` to `S3Backend`.